libvcx/src/credential_def.rs

```diff
@@ -169,7 +208,21 @@ impl CredentialDef {
 
 fn _parse_revocation_details(revocation_details: &str) -> VcxResult<RevocationDetails> {
     serde_json::from_str::<RevocationDetails>(&revocation_details)
-        .to_vcx(VcxErrorKind::InvalidRevocationDetails, "Cannot deserialize RevocationDeltas")
+        .to_vcx(VcxErrorKind::InvalidRevocationDetails, "Cannot deserialize RevocationDetails")
+}
+
+fn _maybe_set_url(rev_reg_def_json: &str, revocation_details: &RevocationDetails, rev_reg_id: &str) -> VcxResult<String> {
```

Comment:
I'd be in favor of extracting the `if let Some(tails_url) = &revocation_details.tails_url {` condition out to the calling code, so this wouldn't have to be called `_maybe_*`.

Reply:
Personally, I wouldn't, I think this way the calling code is much simpler, and the name is still accurate.

Reply:
Okay then, it's not a big deal. Marking resolved